### PR TITLE
Option to refresh browser on set visible.

### PR DESCRIPTION
**Pull request by Etuldan**
Closes https://github.com/obsproject/obs-browser/issues/164

Add an option to enable refresh of browser when source set to visible.

Only locales for en-US and fr-FR

### CODE REVIEW

**Comment by Vainock** [comment]
Please don't translate something here, do that in the OBS Studio Crowdin project: https://crowdin.com/project/obs-studio. Please remove the translation of the string.

**Comment by Etuldan** [comment]
So, I have to close this one, revert on my repo (or start from scratch) the FR-fr translation (I'm native French, that's why), and create a new PR  ?

**Comment by Vainock** [comment]
I understand why do you want to translate it, but it is completey unnecessary because your translations here will not be uploaded to Crowdin and have to be made again there.

I'm not pretty familiar with Git and this stuff, but I think you only have to revert the changes you made in the fr file and commit again, but ask a developer for sure.

**Comment by kkartaltepe** [comment]
There is no need to close this PR. Simply fix up your commit and force push it.

**Comment by Etuldan** [comment]
> 
> 
> There is no need to close this PR. Simply fix up your commit and force push it.

Thx, done :)
(and I fixed also indentation (from spaces to tabs)à

File: obs-browser-plugin.cpp [hunk 1]
```diff
@@ -390,7 +392,10 @@ void RegisterBrowserSource()
 	};
 	info.show = [] (void *data)
 	{
-		static_cast<BrowserSource *>(data)->SetShowing(true);
+		BrowserSource *bs = static_cast<BrowserSource *>(data);
+		if (bs->estart_on_visible)
```
Comment (megalon) [inline obs-browser-plugin.cpp]: I haven't tested this, but this looks like a typo to me.
Shouldn't this be `restart_on_visible`, not `estart_on_visible` ?

Comment (Etuldan) [reply obs-browser-plugin.cpp]: Indeed ...
I'll fix this

File: obs-browser-plugin.cpp [hunk 2]
```diff
@@ -390,7 +392,10 @@ void RegisterBrowserSource()
 	};
 	info.show = [] (void *data)
 	{
-		static_cast<BrowserSource *>(data)->SetShowing(true);
+		BrowserSource *bs = static_cast<BrowserSource *>(data);
+		if (bs->restart_on_visible)
+			bs->Refresh();
```
Comment (WizardCM) [inline obs-browser-plugin.cpp]: The option you're likely looking for is actually `bs->cefBrowser->ReloadIgnoreCache();` if you're going off the #164 request.

**Pull request closed without merging**